site/content/en/docs/Tutorials/allocator-service-go.md

```diff
@@ -265,12 +265,12 @@ status:
 ### 9. Check Game Servers
 Let's make sure that we have one or more Game Servers in a ready state by running this command:
 ```
-kubectl get gs -o=custom-columns=NAME:.metadata.name,STATUS:.status.state,IP:.status.address,PORT:.status.ports
+kubectl get gs
 ```
 
 For a fleet of 2 replicas, you should see 2 Game Servers with a Status of Ready:
 ```
-NAME                     STATUS    IP               PORT
+NAME                     STATUS    IP               PORT                                    
 simple-udp-s2snf-765bc   Ready     35.231.204.26    [map[name:default port:7260]]
```

Comment:
I'm 99% sure the output will change too - i.e. the port only has the first value displayed, rather than the map - so we should adjust those too :+1: 

But this is awesome!

Reply:
@markmandel yes you are right. I have updated the output. Thanks for pointing it out :)